### DISCUSSION TITLE
Confirm before create

### DISCUSSION THREAD

**Discussion by Keko-94** [filamentphp/filament#4111] [Help]
Hi !

I would like to show a confirm alert when I click on the save button of my form (wizard form). (Are you sure ? Yes => keep submiting form / No => cancel submiting form)
If possible I would like to use the sweetalert JS library

Is there any hook to achive that ?

Thanks for your help !

**Comment by danharrin** [comment]
It's possible, but there's not an easy way to just open a modal from a form submit... but I can suggest an easy alternative if this fits:

```
Checkbox::make('confirmed')
    ->label('Are you sure?')
    ->accepted()
    ->required()
    ->dehydrated(false)
```
